serviceinsight/sequence-diagram/index.md

```diff
@@ -1,114 +1,115 @@
+labelled with the message type. If the endpoint received more than one message of that type, the specific message being processed may be seen by hovering over or selecting the processing box.
 
 A message arrow shown leaving a processing box represents an outgoing message that was sent or published while processing the incoming message. These arrows are shown in the order in which those messages were sent or published.
 
-NOTE: If a processing box appears further down the diagram then it was executed later. This means other messages sent by other endpoints may have been processed in the meantime. The size of a processing box and the distance between them is not important.
+NOTE: A processing box appearing further down the diagram indicates that the processing occured later. This means other messages sent by other endpoints may have been processed in the meantime. The size of a processing box and the distance between them is not important.
 
 ![Failed processing](failing-handler.png)
 
-If the processing of a message failed, the processing box is displayed in red with an exclamation mark. If the message has not already been (automatically) retried, it can be retried manually via the context menu.
+If the processing of a message failed, the processing box is displayed in red with an exclamation mark. If the message has not already been (automatically) retried, it can be retried manually via the right-click context menu.
 
 
 ### Events
 
 ![Event](event.png 'width=500')
 
-Events are represented similarly to other messages except that they have dashed lines and a different icon.
+Events are represented similarly to other messages but with dashed lines and a different icon.
 
-NOTE: Each event that is published will appear on the diagram once for each subscriber. This looks like individual messages were sent to each subscriber by the sender regardless of whether Unicast or Multicast routing is used. [Learn more about Message Routing](/nservicebus/messaging/routing.md).
+NOTE: Each event published will appear once for each subscriber, as if individual messages were sent to each subscriber by the sender, regardless of whether Unicast or Multicast routing is used. [Learn more about Message Routing](/nservicebus/messaging/routing.md).
 
 
 ### Loopback messages
 
 ![Loopback](loopback.png)
 
-When an endpoint sends a message to itself this is called a loopback message. On the sequence diagram this is represented by a short arrow that does not connect to another endpoint lifeline and a special icon. As with any other type of message, hovering over or selecting the message will highlight the processing for that message in the lifeline.
+A loopback message is a message that an endpoint sends to itself and is represented by a short arrow with a special icon, that does not connect to another endpoint lifeline. As with any other message, hovering over or selecting the message will highlight the processing for that message in the lifeline.
```

Comment:
It depends whether the concept is called a "loopback" or a "loopback message". If the former, we should change the section header to `### Loopbacks`. If the latter, then this sentence needs to stay as is.